docs/reference/execution-providers/CUDA-ExecutionProvider.md

```diff
@@ -0,0 +1,80 @@
+---
+title: CUDA
```

Comment:
Should this file also live in master ? under docs/execution_providers ? It seems like the other files are present there as well ?

Reply:
this PR is removing them from master: https://github.com/microsoft/onnxruntime/pull/6225
i think this is the official location.

Reply:
Thanks, got it.